src/store.js

```diff
@@ -1,14 +1,14 @@
 var throttle = require('./lib/throttle');
 var toDenseArray = require('./lib/to_dense_array');
-var SimpleSet = require('./lib/simple_set');
+var StringSet = require('./lib/string_set');
 var render = require('./render');
 
 var Store = module.exports = function(ctx) {
   this._features = {};
-  this._featureIds = new SimpleSet();
-  this._selectedFeatureIds = new SimpleSet();
-  this._changedFeatureIds = new SimpleSet();
-  this._deletedFeaturesToEmit = new SimpleSet();
+  this._featureIds = new StringSet();
+  this._selectedFeatureIds = new StringSet();
+  this._changedFeatureIds = [];
```

Comment:
I merged anyway as we can do this in one of the many other PRs coming down the pipe.